### ISSUE TITLE
疑惑？

### ISSUE THREAD

**Issue by sayhelloto1988** [WJJLL/SVD-SSA#1]
你们的svd-logits和inception模型的aux_logits有什么区别呢？像admix这个，使用了aux_logits比没有使用的时候，黑盒上平均会高5%

**Comment by WJJLL** [comment]
请看代码utils下的SVD.py, 我们不是使用aux_logits,  是使用 x = model.Mixed_6e(x)  这一层出来的特征 ，进行svd操作后，再送入后续 mixed-7a等层，得到最后的svd-logits； 而aux_logits 是 x = model.Mixed_6e(x)  这一层出来的特征, 送入 AuxLogits
